README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
+* What experience do you have in developing in frameworks such as Microsoft Active Accessibility (MSAA), Microsoft UI Automation (UIA), or Java Accessibilty Bridge, or Linux Foundation's IAccessible2?
```

Comment:
The question was, are the relevant to front-end development. My answer, absent your suggesting the response to richgilbank was overly broad, is correct. 

You yourself just brought it around to the expected response to the question "What is your approach to accessibility...": adhering to WCAG would be a good thing to here as a leading response.

Specifically for web browsers, the next level of user experience is having knowledge of the assistive technologies (screen readers, magnifiers, etc.) use and how they are used to understand what design patterns or approaches are needed to support them, and what additional tailoring is possible (WAI-ARIA) to include why that can actually be detrimental to your end users (WAI-ARIA override of name, role, state of foundational HTML elements).

Framework is a generic term. Any framework will do. Abstract away! My point is people without the experience assume the framework has accessibility built in or don't know about accessibility so never asked the question, and don't have the experience. They might be smart enough to figure it out, but these are _interview_ questions.

Most of these simple questions have been targeted at front-end development of web browser applications. Although the Access Board's (Section 508) standards for accessibility are drafted and equate many of these accessibility functions currently implemented in web applications in software concepts.

The highest level of understanding of how accessibiltiy affects the end user is understanding the operating systems build in framework (e.g., Microsoft historically using MSAA and now UIA) and, by example, how browser manufacturers use that framework to deliver accessibility to the operating system, or how assistive technologies such as screen readers rely on those same APIs and frameworks to function.

I thought it too advanced to put in this sort of initial questioning, but an developer experienced in accessibility with this background would be able to discern why Firefox's current approach to the operating system accessibility on Windows 8 might fail to work for a screen reader user, even if coded properly. They should also understand how to use the UIA framework to code their own (C, C++, .NET, any language will do) application to insure it is accessible to the operating system and assistive technologies for that platform. By extension they should also understand the general approach to accessibility used, and determine the platform specific accessibility layers indicates iOS is different from Andriod, is different from Linux+Window manager, is different from Windows (MSAA for earlier versions, UIA since Windows 7).

I hope that helps explain my perspective. Written responses isn't always the easiest medium to explain nor provide emotional intent to the response, so please forgive me if my attempt to clarify seems curt. I only wanted to respect your time and try and be clear. I sincerely thank you for taking the time to be critical of my suggestions, as you've also given me a chance to consider how to better introduce the topic to people like yourself that may have the same concerns. Please continue to help me make this better!